Explorer/Assets/Scripts/ECS/StreamableLoading/GLTF/GltFastDownloadProvider.cs

```diff
@@ -49,9 +58,17 @@ public async Task<IDownload> RequestAsync(Uri uri)
             {
                 webRequest.downloadHandler = new DownloadHandlerBuffer();
 
-                try { await webRequest.SendWebRequest().WithCancellation(new CancellationToken()); }
-                catch { throw new Exception($"Error on GLTF download ({targetGltfOriginalPath} - {uri}): {webRequest.downloadHandler.error} - {webRequest.downloadHandler.text}"); }
+                try
+                {
+                    await webRequest.SendWebRequest().WithCancellation(new CancellationToken());
+                }
+                catch
+                {
+                    if (isBaseGltfFetch) acquiredBudget.Release();
```

Comment:
This section could go into finally

Reply:
Tackled at https://github.com/decentraland/unity-explorer/pull/2430/commits/6cef542ac352b828b0373564e4cc1b909245065c